src/common/components/paymentMethods/creditCardForm/creditCardForm.tpl.html

```diff
@@ -71,7 +71,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Credit Card Paym
       <div class="col-sm-4" ng-hide="$ctrl.paymentMethod">
```

Comment:
I think you can make this use `ng-if` to remove it from the DOM and prevent its validation from being run.